src/main/java/uk/gov/hmcts/reform/hmc/model/linkedhearinggroup/LinkedHearingGroupResponses.java

```diff
@@ -0,0 +1,15 @@
+package uk.gov.hmcts.reform.hmc.model.linkedhearinggroup;
+
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.util.List;
+
+@Data
+@NoArgsConstructor
+public class LinkedHearingGroupResponses {
```

Comment:
Perhaps to avoid some very similar names.. could call this class `GetLinkedHearingGroupResponse`, and the `hearingsInGroup` could be a `List<LinkedHearingDetails>` (stolen from the spec!).